### PR TITLE
v | added vweb

### PR DESCRIPTION
**Pull request by koplenov**
vweb - A simple yet powerful web server with built-in routing, parameter handling, templating, and other features. The [gitly](https://gitly.org/) site is based on vweb.

site: https://modules.vlang.io/vweb.html
source: https://github.com/vlang/v/tree/master/vlib/vweb
language: https://vlang.io/

### CODE REVIEW

File: v/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+FROM thevlang/vlang
+
+WORKDIR /app
+
+{{#files}}
+  COPY '{{source}}' '{{target}}'
+{{/files}}
+
+RUN v . -o server
```
Comment (waghanza) [inline v/Dockerfile:9]: do you not have any build action ? I mean compilation ....

File: v/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,11 @@
+FROM thevlang/vlang
```
Comment (waghanza) [inline v/Dockerfile:1]: no tag on the docker image ?

File: v/Dockerfile (hunk 1 above)
Comment (koplenov) [reply v/Dockerfile:9]: v . - is build action.. 😅
The first time I failed to set up a two-stage image. I'll try again later

File: v/Dockerfile (hunk 2 above)
Comment (koplenov) [reply v/Dockerfile]: Ready

File: v/Dockerfile (hunk 1 above)
Comment (koplenov) [reply v/Dockerfile]: Check it out now

Comment (spytheman) [reply v/Dockerfile]: > v . - is build action.. 😅

Why not `v -prod .` ?

**Review by waghanza — APPROVED** [review]

**Pull request merged**